src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -324,6 +337,9 @@ class AdequacyPatchRuntimeData
             extremityAreaMode.push_back(link.with->adequacyPatchMode);
         }
     }
+
+private:
+    std::set<std::pair<int, int>> csrTriggered_;
```

Comment:
Maybe there's a better structure for insert/find execution times, but I can't think of any.

Reply:
See [this discussion](https://gist.github.com/flomnes/5614997a44660ab7b8836ef3cd6497be)